### ISSUE TITLE
Request: Simple calendar

### ISSUE THREAD

**Issue by PaulKinlan** [WebUtils/Platform#15]
It is just a calendar that lists the dates, lets you page through months so you can quickly see when dates are.  It won't include any events or other calendar features.

**Comment by mahemoff** [comment]
Can you please prefix these with "Request"

**Comment by mahemoff** [comment]
I agree it should be simple without much details but should of course highlight today's date.
